docs/concepts/controllers/statefulsets.md

```diff
@@ -0,0 +1,171 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+---
+
+{% capture overview %}
+**StatefulSets are a beta feature in 1.5. This feature replaces the 
+Pet Sets feature from 1.4. Users of Pet Sets are referred to the 1.5 
+[Upgrade Guide](/docs/tasks/stateful-set/upgrade-from-petsets-to-stateful-sets/)
+for further information on how to upgrade existing Pet Sets to StatefulSets.**
+
+A StatefulSet is a Controller that provides a unique identity to its Pods. It provides
+guarantees about the ordering of deployment and scaling.
+{% endcapture %}
+
+{% capture body %}
+
+### When to Use a StatefulSet
+StatefulSets are valuable for applications that require one or more of the 
+following.
+
+* Stable, unique network identifiers.
+* Stable, persistent storage.
+* Ordered, graceful deployment and scaling.
+* Ordered, graceful deletion and termination.
+
+In the above, by stable, we mean persistent across Pod (re) schedulings.
+As it is generally easier to manage, if an application doesn't require any of 
+the above guarantees, and if it is feasible to do so, it should be deployed as 
+a set of stateless replicas. Before deciding to use a StatefulSet, you should 
+be certain that a [Deployment](/docs/user-guide/deployments/) or a
+[ReplicaSet](/docs/user-guide/replicasets/) is not better suited to your needs.
+
+### Limitations
+* StatefulSet is a beta resource, not available in any Kubernetes release prior to 1.5.
+* As with all alpha/beta resources, it can be disabled through the `--runtime-config` option passed to the apiserver.
```

Comment:
Avoid pronouns. "It" -> "StatefulSet".

Avoid passive voice. "It can be disabled" -> "You can disable StatefulSet through the `--runtime-config` option that you pass to the Kubernetes apiserver."